libseccomp/src/notify.rs

```diff
@@ -236,6 +273,12 @@ impl ScmpNotifResp {
     /// * `val` - Return value for the syscall that created the notification
     /// * `error` - An error code
     /// * `flags` - Userspace notification response flag
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
```

Comment:
`no_run` is better.

Reply:
changing it to `no_run` causes failure on running `cargo test --doc`.
```shell
test libseccomp/src/notify.rs - notify::ScmpNotifResp::new (line 278) - compile ... FAILED

failures:

---- libseccomp/src/notify.rs - notify::ScmpNotifResp::new (line 278) stdout ----
error[E0433]: failed to resolve: use of undeclared type `ScmpNotifResp`
 --> libseccomp/src/notify.rs:279:12
  |
3 | let resp = ScmpNotifResp::new(req.id, val, error, flags);
  |            ^^^^^^^^^^^^^ use of undeclared type `ScmpNotifResp`
  |
help: consider importing this struct
  |
2 + use libseccomp::ScmpNotifResp;
  |

error[E0425]: cannot find value `req` in this scope
 --> libseccomp/src/notify.rs:279:31
  |
3 | let resp = ScmpNotifResp::new(req.id, val, error, flags);
  |                               ^^^ not found in this scope

error[E0425]: cannot find value `val` in this scope
 --> libseccomp/src/notify.rs:279:39
  |
3 | let resp = ScmpNotifResp::new(req.id, val, error, flags);
  |                                       ^^^ not found in this scope

error[E0425]: cannot find value `error` in this scope
 --> libseccomp/src/notify.rs:279:44
  |
3 | let resp = ScmpNotifResp::new(req.id, val, error, flags);
  |                                            ^^^^^ not found in this scope

error[E0425]: cannot find value `flags` in this scope
 --> libseccomp/src/notify.rs:279:51
  |
3 | let resp = ScmpNotifResp::new(req.id, val, error, flags);
  |                                                   ^^^^^ not found in this scope

error: aborting due to 5 previous errors

Some errors have detailed explanations: E0425, E0433.
For more information about an error, try `rustc --explain E0425`.
Couldn't compile the test.

failures:
    libseccomp/src/notify.rs - notify::ScmpNotifResp::new (line 278)

test result: FAILED. 51 passed; 1 failed; 3 ignored; 0 measured; 0 filtered out; finished in 1.30s

error: doctest failed, to rerun pass `-p libseccomp --doc`

```

Reply:
Yes, ideally you need to make the code compilable like other example codes.
But, considering this again carefully, users will call `new_val` or `new_error` instead of this `new` api (the document also recommends they should do so). Hence, it's okay with what it is. Could you send a new commit to include the other comment?

Reply:
@ManaSugi 
I have pushed the commit with requested changes.